india_compliance/gst_india/api_classes/base.py

```diff
@@ -12,6 +12,10 @@
 BASE_URL = "https://asp.resilient.tech"
 
 
+class GatewayTimeoutError(Exception):
+    pass
```

Comment:
this is not the best place to define an exception, look at `frappe.exceptions` & `erpnext.exceptions`.

Reply:
consider specifying default error message, like so: https://stackoverflow.com/questions/49224770/default-message-in-custom-exception-python